wiki/Ranking_criteria/Metadata/en.md

```diff
@@ -0,0 +1,266 @@
+    - `(Long)` -> `(Long Ver.)`
+  - Exceptions would be for when the length / version marker is so stylised it is considered part of the title, such as `Pippiquest (Pippi x Mocha Romantic Movie Remix Edition)`
+
+#### Allowances
+
+- **Alternative casing for markers may be used if the rest of the song title is stylised to fit the formatting.**
+- **Live performances may add a special marker, such as `(Live Ver.)`.** Descriptive markers like `(2020 Tour Live Ver.)` can also be used.
+- **Marker additions may be ignored or a custom marker may be used on a case-by-case basis if the standard markers are misleading.** Hold a discussion to determine the marker in this case and post the result publicly. 
+
+## Source
+
+### Rules
+
+- **The `Source` field must be used, if the song...**
+  - **directly originates from or is tied to a piece of media (video games, TV series, etc.), except for albums and hosting websites.**
+  - **is a remix, arrangement or cover of a song from or tied to a piece of media, except for albums and hosting websites.**
+  - **was specifically created for osu!, such as [osu! originals](/wiki/osu!_originals).** Any other song *must not* use `osu!` as the source field.
+  - **was created for a specific event, such as a tournament like the `osu! World Cup` or concerts.**
+- **If a track contains or remixes multiple songs that are not all from one shared source, the field must be left blank and the sources added to tags.**
+- **The Source field must be precise.** Use the most specific source instead of general series or project names, unless multiple sources within a series apply.
+
+### Guidelines
+
+- **If a track...**
+  - **was first released and later featured or tied to a piece of media, using the source field is optional.**
+  - **has been featured in multiple pieces of media, any option can be used as the source.**
+- **Website names are only valid sources, if the song...**
+  - **and website are tied to specific cultural phenomena such as `Newgrounds`, etc.**
+  - **was composed as a website theme or background song.**
+
+## Tags
+
+### Rules
+
+- **Tags must be related to the beatmap**, such as describing the style, song, storyboard, video, or background content. Misleading tags must be avoided.
+- **Names with spaces between single characters like `-[M o c h a]-` need to be tagged as `-[M_o_c_h_a]-`**
+- **Tags must include the following items when applicable:**
+  - **[Guest difficulty](/wiki/Beatmap/Guest_difficulty) creators, storyboarders, skinners and hitsounders.**
+  - **`Featured Artist`, if the track is listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).** You must not use the tag if the song is not in the featured artist catalogue.
+  - **At least one song genre and one language tag.** 
+    - For instrumental tracks, `instrumental` is the language tag.
+    - For tracks in artificially created languages add `conlang` to tags and use its name as the language tag.
```

Comment:
```suggestion
    - For tracks in artificially created languages add `conlang` to tags as the language tag. If the conlang has its own name, add that to tags as well.
```

what "its name" refers to was unclear and clunky. how about this?

Reply:
i have never heard of an unnamed conlang, cuz the nature of conlangs is to make up names for stuff 

that said i think that may be better?

Reply:
how about just `and use the conlang's name as the language tag.` instead